### PR TITLE
Messages when register

### LINKED ISSUE #138: Add success / error messages when logging in or registering with an invite code.
This should include some kind of error / success messages like

You have successfully join X project.
Your invite link was invalid.
X project has been archived and is no longer available to join.

This should happen when both registering and logging in.

### PR DESCRIPTION
**Pull request by adamchlebek**

### CODE REVIEW

**Review by dyl10s — CHANGES_REQUESTED** [review]
I did not get any message when I registered, I also think there should be a specific message for when you try and join an archived project.

**Comment by jwaltt55** [comment]
We should probably have a message that indicates a user is already part of a project if they try to join one they've already joined, as of right now I just get a message saying that joining was successful.

**Comment by jwaltt55** [comment]
This is also kind of a nitpicky thing but I think it'd be more helpful to a user to have the error message for an invalid invite code to be something a little more specific. "Invite code hasn't been found" would probably seem kind of vague to just the average user I think.

**Comment by jwaltt55** [comment]
It looks like if you use the invite code with the login/register URLs while already logged in, you'll just get a "Invite code is invalid" message every time if you can't join the project, instead of the messages telling the user that they've already joined a project or that the project is archived. We probably want the error messages to be the same for both when the user has to login/register, and when they use the invite code while already logged in.

File: Frontend/src/app/views/auth/login/login.component.ts [hunk 1]
```diff
@@ -78,9 +78,11 @@ export class LoginComponent implements OnInit {
               if(response.message == "Added User to Project"){
                 this.toastrService.success("You have been added to a project", "Success");
               }else if(response.message == "Project not found"){
-                this.toastrService.danger("Invite code has not been found", "Error")
+                this.toastrService.danger("Invite code is not valid", "Error")
               }else if (response.message == "Unable to add to Archived Project"){
                 this.toastrService.danger("Unable to add user to Archived Project", "Error");
+              }else if (response.message == "User already in project"){
+                this.toastrService.danger("User is already assigned to this project", "Error");
```
Comment (dyl10s) [inline Frontend/src/app/views/auth/login/login.component.ts]: I feel like this wording is a bit off, we are talking directly to the user this seems more of like a backend logging message.

**Review by dyl10s — DISMISSED** [review]

**Comment by jwaltt55** [comment]
Just a couple small things:

There's a typo for when you first login with an invite code (should be "a part" not "apart").

Also just in general I think the messages for when you use the invite code with a login/register endpoint when you're already logged in and when you're not logged in/haven't made an account yet should be the same, it's good that they're more specific now for when you're already logged in but they're kinda inconsistent with the messages you get if you're not logged in yet.

I'm also still getting the "user was successfully added to a project" message instead of the "you're already a part of this project" message sometimes, usually when I create a project and the immediately try to join it with an invite code a second time.

**Review by jwaltt55 — APPROVED** [review]

**Pull request merged**